### ISSUE TITLE
The system magnifier can't follow the text input cursor in editor still

### ISSUE THREAD

**Issue by JiayangShen** [microsoft/vscode#83276]
- VSCode Version: 1.39.2
- OS Version:  Windows_NT x64 10.0.17763

Steps to Reproduce:
1. Press Win + R shortcut key to open the "Run Dialog", and input "magnify" to open the magnifier tool in the windows system. Then setting the magnifier to follow mouse pointer, follow keyboard focus, and follow text input cursor.
2. Open vscode, new a blank document, input some long text. You will find that the magnifier can't follow the text input cursor in editor.

It is the same issue as #7394, #7394 is closed but the issue remains, so I have to re-report by open a new same one.

I find something may can help, it is:

1. In vscode editor, open the developer tools.
2. In Elements panel, search "textarea", findout the element `<textarea data-mprt="6" class="inputarea" wrap="off" autocorrect="off" autocapitalize="off" autocomplete="off" spellcheck="false" aria-label="现在无法访问编辑器。按 Alt+F1  显示选项。" role="textbox" aria-multiline="true" aria-haspopup="false" aria-autocomplete="both" style="font-size: 1px; line-height: 19px; top: 0px; left: 68px; width: 1px; height: 1px;"></textarea>`.
3. Remove the style `font-size: 1px;` of the textarea element in devtool's Styles panel.
4. input some text in the vscode editor, the magnifier works!
![vscode](https://user-images.githubusercontent.com/8371875/67552848-708f8e00-f73e-11e9-87b4-d73e6c6d8c15.png)

**Comment by isidorn** [comment]
Thanks for filling this.
However none of the other users have complained. I somewhat believe that is due to your Windows_NT x64 OS. On Windows 10 I can not reproduce this issue

**Comment by alexandrudima** [comment]
The magnifier also works fine for me on Windows 10 . It follows the cursor fine.

I would like to be able to reproduce before doing a code change, I don't understand why the fontSize would be relevant for the magnifier without at least reproducing.

@JiayangShen are you on Windows 10?

**Comment by JiayangShen** [comment]
> The magnifier also works fine for me on Windows 10 . It follows the cursor fine.
> 
> I would like to be able to reproduce before doing a code change, I don't understand why the fontSize would be relevant for the magnifier without at least reproducing.
> 
> @JiayangShen are you on Windows 10?
Yes, it is on windows 10 (Windows_NT x64 10.0.17763). And I find that except remove the font-size, if set font-size bigger then 1.67px, the magnifier also works. emmm...it is really too strange!

**Comment by JiayangShen** [comment]
I also tried the insiders version, it has the same issue.

**Comment by alexandrudima** [comment]
@JiayangShen I would still like to reproduce, so I can check that the code change makes a difference... Are you perhaps on a high DPI screen or otherwise using this zoom option in Windows or the `window.zoomLevel` setting?

![image](https://user-images.githubusercontent.com/5047891/67568202-dc1c3000-f72b-11e9-8d83-7cb30fd3fe5f.png)

**Comment by JiayangShen** [comment]
@alexandrudima thanks for your idea! After I set the editor's `window.zoomLevel` value to 0.1, the magnifier works! the default value 0 does not work. Or, press ctrl and use mouse wheel to zoom out the editor, the magnifiter works too. if zoom in and let the font-size small enough, magnifier will not work.

**Comment by sbatten** [comment]
I verified by comparing behavior of the following steps in stable and insiders:
1. Set zoom level to -8
2. Open an untitled file and type many lines of text
3. Navigate the document up and down using the editor cursor

Stable: magnifier stops tracking cursor
Insiders: magnifier continues to track cursor

**Comment by anthonykozak** [comment]
> I verified by comparing behavior of the following steps in stable and insiders:
> 
> 1. Set zoom level to -8
> 2. Open an untitled file and type many lines of text
> 3. Navigate the document up and down using the editor cursor
> 
> Stable: magnifier stops tracking cursor
> Insiders: magnifier continues to track cursor

Same here, I will have to use the Insiders version then. Thanks for the tips!
Also the text following feature is not working in the Terminal view, at all with both VS versions on PowerShell and CMD. It's sad cause I'm using the console a lot. Should I open another issue for that ?
Thank you!

**Comment by anthonykozak** [comment]
After applying my settings to the Insiders version, I noticed the same issues, so I tried to find which particular setting was breaking it.
And finally removing the line "editor.accessibilitySupport": "on",
inside C:\Users\<USER>\AppData\Roaming\Code\User\settings.json

made the trick, so I guess this mode should be buggy.

**Comment by isidorn** [comment]
This fix is in the vscode stable also, so you do not have to switch to insiders.
`editor.accessibilitySupport: "on"` should not have an influence on this feature.
If that is no the case, can you please create a new issue with minimal reproducable steps so we look into it. Thanks
